tests/3-continuous_manage_queues_validation.bats

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bats
+
+load libs/shared_setup
+
+setup () {
+    export WORKSPACE=$gitdir
+
+    # Some dummy variables just for the sake of the letting the script proceed through the validation.
+    export jiraclicmd=jiracli
+    export jiraserver=https://tracker.moodle.org
+    export jirauser=jirauser
+    export jirapass=jirapass
+}
+
+@test "tracker_automations/continuous_manage_queues/continuous_manage_queues.sh: Current date < release date" {
+    export releasedate=$(date -d "+10day" +%Y-%m-%d)
+    source $PWD/tracker_automations/continuous_manage_queues/lib.sh
+
+    run run_param_validation $releasedate
+
+    # Assert result.
```

Comment:
I would add `assert_success` (maybe because I always add one of success/failure assertions). That way, if some error happens later, after the message has been printed, it will be detected.

Reply:
Ooops, yes. I added `assert_success`.